src/components/LanguageSelector/__tests__/LanguageSelector.test.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import LanguageSelector from '..';
+
+describe('LanguageSelector component', () => {
+  it('renders correctly', () => {
+    render(<LanguageSelector />);
+    const list: HTMLUListElement = screen.getByRole('list', { hidden: true });
+    const listItems: HTMLLIElement[] = screen.getAllByRole('listitem', {
+      hidden: true,
+    });
+    expect(list).toBeInTheDocument(); // We can't rely on snapshots, because the array of locales is dynamic structure
```

Comment:
The array actually is static, and I think if someone enables/disables a language, they should update the snapshots 👀

Reply:
Ok, makes sense